rasa/nlu/test.py

```diff
@@ -1909,7 +1913,7 @@ def compare_nlu(
                 model_path = os.path.join(get_model(model_path), "nlu")
 
                 output_path = os.path.join(model_output_path, f"{model_name}_report")
-                result = run_evaluation(
+                result = run_evaluation_async(
```

Comment:
`run_evaluation_async` doesn't exist. Did you mean this?
```suggestion
                result = run_evaluation(
```